templates/blogpost.md.ejs

```diff
@@ -14,7 +14,7 @@ const RULE_REGEX = new RegExp(`\`?\\b(${ruleList.join("|")})\\b\`?`, "g");
 
 function linkify(line) {
     return line
-        .replace(/#(\d+)/g, "[#$1](https://github.com/eslint/eslint/issues/$1)")
+        .replace(/([^\s/]+[/][^\s/]+)?#(\d+)/g, (_, repo, num) => `[${repo || ""}#${num}](https://github.com/${repo || "eslint/eslint"}/issues/${num})`)
```

Comment:
Sorry, it's tough to read those RegExp. Since the first group is optional, wouldn't `repo` version contain ticket number if repo is not present?

Reply:
No, `repo` will be `undefined` if repo is not present.